src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.service.impl;
+
+import java.util.ArrayList;
+import mate.academy.dao.ShoppingCartDao;
+import mate.academy.dao.TicketDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.security.EntityNotFoundException;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private static ShoppingCartDao shoppingCartDao;
+    @Inject
+    private static TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        final ShoppingCart shoppingCart = getByUser(user);
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
+        shoppingCart.getTickets().add(ticket);
```

Comment:
it's better to declare variable in place where you are using it

```suggestion
        ShoppingCart shoppingCart = getByUser(user);
        shoppingCart.getTickets().add(ticket);
```